src/components-v2/VaultApyInformation/index.tsx

```diff
@@ -323,11 +331,13 @@ const VaultApyInformation = ({ open, onClose, boost, vault, projectedBoost }: Pr
             Fees have been deducted from APY estimates. See the vault details page for fee information.
           </Typography>
         </Grid>
-        <Grid item className={classes.button}>
-          <Button color="primary" variant="contained" fullWidth onClick={handleGoToVault}>
-            GO TO VAULT
-          </Button>
-        </Grid>
+        {removeGoToVaultButton !== true && (
```

Comment:
Any specific reason for using `removeGoToVaultButton !== true`, we can simply use `!removeGoToVaultButton`

Reply:
thanks